packages/components/src/Autocomplete/Autocomplete.test.tsx

```diff
@@ -71,7 +71,7 @@ describe('Autocomplete', () => {
     expect(screen.queryByRole('option')).toBeNull();
   });
 
-  it('Props:multiple=false: в инпут сетится label после выбора option', async () => {
+  it('В инпут сетится label после выбора option при multiple=true', async () => {
```

Comment:
```suggestion
  it('В инпут сетится label после выбора option', async () => {
```

Компоненту не пробрасывается multiple

Reply:
> it('В инпут сетится label после выбора option при multiple=true', async () => {
>   it('В инпут сетится label после выбора option', async () => {

Много раз пришлось прописывать multiple=true, к этой строке писал уже почти на автомате. Пофикшено =)